Droidcon-Boston/app/src/main/res/values/strings.xml

```diff
@@ -42,4 +42,5 @@
     <string name="str_agenda_detail_time">%1$s - %2$s</string>
     <string name="close">Close</string>
     <string name="droidconbos_twitter_handle">droidconbos</string>
+    <string name="no_bookmarked_items">No talks added to Agenda\nfor this day</string>
```

Comment:
should be "No talks added to My Schedule\nfor this day", as per one of the bugs filed in trello.